src/index.js

```diff
@@ -134,11 +134,18 @@ var VERSION = '__VERSION__';
  */
 
 /**
- * @typedef {object} threeDSecureOptions _Deprecated_ If the `threeDSecureOptions` passed into the create call is an object, you may set the `amount` to verify with 3D Secure. However, it's recomended that you pass `true` instead of a configuration object and do all 3D Secure configuration in the {@link Dropin#requestPaymentMethod|requestPaymentMethod options}.
```

Comment:
random q: does this essentially mean the `threeDSecureOptions` parameter has been "un-deprecated" here?

Reply:
Yep! We are "un-deprecating" it since now merchants can pass in a `cardinalSDKConfig` object here.

Reply:
yeah, originally, all the 3ds config stuff was put in an object at the time of creation (at the time, just `amount`). With 3ds 2, most of the parameters are things you collect _as_ you check out, so you don't have them at the point of creating Drop-in, so we elected to pass those values at the time you call `requestPaymentMethod`.

With this addition, we need it at the time of creation, so we're un-deprecating the object as a whole, and only deprecating the `amount` field.